bump_version.py

```diff
@@ -2,7 +2,7 @@
 import io
 import pathlib
 
-import parver
+import parver  # type: ignore
```

Comment:
Wouldn't it be better to set `ignore_missing_imports`? Anyway, doesn't matter that much.

Reply:
I would do that if `mypy` had configuration in `pyproject.toml`, but it seems like it doesn't, and I don't want to add a new file for this.

If `parver` is willing to drop Python 2, I may offer to add type stubs to the package at some point.